packages/twenty-server/src/engine/core-modules/cache-storage/services/cache-storage.service.ts

```diff
@@ -107,7 +107,7 @@ export class CacheStorageService {
 
     do {
       const result = await redisClient.scan(cursor, {
-        MATCH: scanPattern,
+        MATCH: `${this.namespace}:${scanPattern}`,
```

Comment:
cache-storage service should respect its namespace. Note that this change the cache flush command that will only be able to flush engine:workspace keys